modules/markdown/markdown_test.go

```diff
@@ -50,18 +52,18 @@ func TestMarkdown(t *testing.T) {
 					So(string(RenderIssueIndexPattern([]byte(testCases[i]), urlPrefix, metas)), ShouldEqual, testCases[i+1])
 				}
 			})
-			Convey("It should not render issue without leading space", func () {
+			Convey("It should not render mention without leading space", func() {
```

Comment:
You're right, that should be an issue mention.  A mention could be a commit mention, an issue mention, or a username mention.